pyvista/core/pointset.py

```diff
@@ -392,11 +392,17 @@ def faces(self, faces):
         if isinstance(faces, CellArray):
             self.SetPolys(faces)
         else:
+            # faster to mutate in-place if array is same size?
```

Comment:
This comment gives me concern :P Is it a TODO? Is it a note? Is it something else? I don't really get it.

Reply:
This was a TODO to look into when we had considered attempting to mutate rather than assign with SetPolys.  Let's consider looking into this later, but leave it outside of this PR.